GameEngine/Characters/Character.cs

```diff
@@ -27,19 +27,24 @@ public Character(string name, int hp)
         public virtual void Turn(GameSourceData gameData)
         {
         }
-
-        public virtual void Attack(Character attackingCharacter, GameSourceData gameData)
+        public List<Character> GetCharacters(GameSourceData gameData)
```

Comment:
Let's make this private for now and write a TODO: comment to move this method over to gamestate when we can.